packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -166,6 +186,10 @@ export class AllocationReceiptCollector implements ReceiptCollector {
           },
         )
 
+      this.metrics.receiptsToCollect.set(
+        { allocation: receipts[0].allocation },
+        receipts.length,
+      )
```

Comment:
In this case, why do we use the first receipt allocation?
Is it because we expect them to have been previously filtered with unique allocation ids?

Reply:
Yes I think receipts in the same batch are of the same allocation id:
The receipts are filtered with allocation id when grouped into a batch ([line](https://github.com/graphprotocol/indexer/blob/9cf1468afcefc97f91e9e332163cbffdb76ab750/packages/indexer-common/src/allocations/query-fees.ts#L585) of code) and pushed to `receiptsToCollect`, and then the batches are popped off for collecting ([line](https://github.com/graphprotocol/indexer/blob/9cf1468afcefc97f91e9e332163cbffdb76ab750/packages/indexer-common/src/allocations/query-fees.ts#L235))